src/messages/initialize.cc

```diff
@@ -25,6 +25,47 @@
 namespace ccls {
 using namespace llvm;
 
+const char * SEMANTIC_TOKENS[] = {
+  "unknown",
+
+  "file",
+  "module",
+  "namespace",
+  "package",
+  "class",
+  "method",
+  "property",
+  "field",
+  "constructor",
+  "enum",
+  "interface",
+  "function",
+  "variable",
+  "constant",
+  "string",
+  "number",
+  "boolean",
+  "array",
+  "object",
+  "key",
+  "null",
+  "enumMember",
+  "struct",
+  "event",
+  "operator",
+  "typeParameter",
+  "typeAlias", //252 => 27
+  "parameter",
+  "staticMethod",
+  "macro"
+};
+
+const char * SEMANTIC_MODIFIERS[] = {
```

Comment:
clang-format should make it `const char *SEMANTIC_MODIFIERS[] = `

The preferred style is `kSemanticModifiers`